### PR TITLE
chore(deps): update dependency teeny-request to v5

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

**Comment by DominicKramer** [comment]
@kjin To remove `@types/request` the debug agent needs to be updated to use `teeny-request` instead of `request`.  That should be done in a separate PR and is being tracked by #749.

File: test/test-controller.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ import * as nock from 'nock';
 import {Debug} from '../src/client/stackdriver/debug';
 import {Debuggee} from '../src/debuggee';
 import * as stackdriver from '../src/types/stackdriver';
-import * as r from 'request'; // types only
+import * as t from 'teeny-request';
```
Comment (kjin) [inline test/test-controller.ts]: Merge with below import.

**Comment by kjin** [comment]
I'm pretty sure this library already uses `teeny-request`, all you need to do is remove two references to `request` types.

File: test/test-controller.ts (hunk 1 above)
Comment (DominicKramer) [reply test/test-controller.ts]: Done

**Pull request merged**